src/adapter/src/flags.rs

```diff
@@ -42,6 +42,7 @@ pub fn compute_config(config: &SystemVars) -> ComputeParameters {
         enable_operator_hydration_status_logging: Some(
             config.enable_compute_operator_hydration_status_logging(),
         ),
+        enable_lgalloc_eager_reclamation: Some(config.enable_columnation_lgalloc()),
```

Comment:
```suggestion
        enable_lgalloc_eager_reclamation: Some(config.enable_lgalloc_eager_reclamation()),
```

Reply:
I suspect this is why the flag doesn't work :)

Reply:
Ups, thanks for spotting this! Fixed.